### PR TITLE
update: App General Styles

### PR DESCRIPTION
**Pull request by ghost**
Template basis

### CODE REVIEW

File: backend/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6929 @@
+{
```
Comment (fernandogruning) [inline backend/package-lock.json:1]: Remove these `package-lock.json`s. The project should only have `yarn.lock`s

File: client/src/App.css [hunk 2]
```diff
@@ -1,38 +1,36 @@
 .App {
-  text-align: center;
+  margin: auto;
+  width: 35%;
```
Comment (fernandogruning) [inline client/src/App.css:3]: Why the percentages? They don't scale so well to either large or small screens

File: client/src/App.test.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+test("renders header and body", () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  const header = screen.getByText(/To Do/i);
+  const body = screen.getByTestId("body");
+
+  expect(header).toBeInTheDocument();
+  expect(body).toBeInTheDocument();
 });
```
Comment (fernandogruning) [inline client/src/App.test.js:11]: Excellent!! 👍

File: client/src/index.css [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
 body {
+  height: 100%;
+  width: 100%;
+}
+
+#root {
+  height: 100%;
+  width: 100%;
+  display: flex;
+}
```
Comment (fernandogruning) [inline client/src/index.css:11]: I wouldn't do this, as `#root` is a React-specific element, so this app wouldn't work if you would port it to another framework.

Comment (fernandogruning) [reply client/src/index.css:11]: Remove this styles and change `.App` so it behaves as expected.

File: client/src/App.css [hunk 5]
```diff
@@ -1,38 +1,36 @@
...
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+  background-color: var(--primary);
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  font-size: calc(10px + 1.5vmin);
```
Comment (fernandogruning) [inline client/src/App.css:25]: I would keep it simple and just use rems or even pixel sizes

**Review by fernandogruning — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 2 above)
Comment (Enmanuellperez98) [reply client/src/App.css:3]: So, what kind of measure you expect? vw?

**Review by fernandogruning — APPROVED** [review]

**Pull request merged**